wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/CommandPartElement.scala

```diff
@@ -5,4 +5,4 @@ sealed trait CommandPartElement extends TaskSectionElement
 object CommandPartElement {
   final case class StringCommandPartElement(value: String) extends CommandPartElement
   final case class PlaceholderCommandPartElement(expressionElement: ExpressionElement) extends CommandPartElement
-}
\ No newline at end of file
+}
```

Comment:
This file has a 9th line when I look at it in my IDE.

Reply:
Yeah, github is highlighting this as no longer being a bad EoF so I think we're all good